UKHomeOffice/sgmr-service
# NMSW-1381 - Fix pleasure craft page hrs

## AC

There's an additional HR line showing in the upper section

Refactor code to work more like the people page so we don't need to use the special PageContainer file for it (this should remove the rogue hr as well)

Deleted files that are no longer needed

----

## To test

- run app
- go to pleasure craft (PC) page
- > there should not be an extra hr element
- > the button should no longer be a 'start' button (this did not match the other page buttons)
- click button to add PC
- complete the form
- > You should be able to save a PC
- click to edit the PC
- > You should be able to edit and save the changes
- click to edit the PC again
- delete the PC
- > you should see a notification that the PC is deleted

----

## Notes

- The app should run as normal, even with the files deleted as I'm pretty sure that we didn't use any of them anymore
